src/helpers/update-loan.js

```diff
@@ -1,5 +1,6 @@
 const Loan = require('@lulibrary/lag-alma-utils/src/loan')
 
+// Helper method for SNS Handler Lambdas to write new Loan data to the cache
```

Comment:
I'm not sure of this pattern a `Loan.update()` method seems more appropriate. Similar for other areas.

Reply:
I can't make my mind up on whether or not the whole update process of populating the data and setting the expiry date is too specific to have in the Loan class or not. I think the code was clearer when these update methods were in the same files as the Lambdas, but then they end up being repeated.

Reply:
In my mind the Loan class should have a set of methods which should follow the "one thing well" philosophy. If the class was ever to be used elsewhere, it doesn't seem right to have methods on the class which provide functionality only to one specific use case. 

However, this means that the Lambdas then have to perform multiple operations on the Loan class to write the correct data to the cache, and it seems better to separate that code out from the Lambda handler, which leads to the current situation.

Reply:
I suppose that it's better to have a Loan#update method which both populates the class with input data and then saves it to the cache, and a Loan#addExpiryDate method which adds an expiry date and saves to the cache, however calling these sequentially leads to pointlessly waiting for the database queries to execute.